src/source-pcap-file-helper.c

```diff
@@ -40,6 +40,13 @@ void CleanupPcapFileFileVars(PcapFileFileVars *pfv)
             pfv->pcap_handle = NULL;
         }
         if (pfv->filename != NULL) {
+            if(pfv->shared->should_delete) {
+                SCLogDebug("Deleting pcap file %s", pfv->filename);
+                if(remove(pfv->filename) != 0) {
```

Comment:
Same

Reply:
Why not using unlink here ? Better to limit to file if deleting a directory is not planned.